### PR TITLE
Leap/dashboard

### PR DESCRIPTION
**Pull request by simleap**
screenshots https://goo.gl/A5dJv3

### CODE REVIEW

File: app/models/user.rb [hunk 1]
```diff
@@ -3,7 +3,7 @@ class User < ApplicationRecord
 
   # Include default devise modules. Others available are:
   # :confirmable, :lockable, :timeoutable and :omniauthable
-  devise :invitable, :database_authenticatable, :registerable,
+  devise :invitable, :registerable, :database_authenticatable,
```
[the remark this answers is not in the document]
Comment (simleap) [reply app/models/user.rb]: we need registerable for user change password or information, i have disabled sign up and enable only edit user in next commit.

**Comment by simleap** [comment]
@samnang, i have refactored and add missing tests.

**Comment by simleap** [comment]
@samnang, what is the different between travis-ci/pr and travis-ci/push?

**Comment by samnang** [comment]
> what is the different between travis-ci/pr and travis-ci/push?	

@simleap https://d.pr/i/8tgVXS

I might probably because Dave and I worked on user management branch, and we still have some failing tests :)

**Comment by samnang** [comment]
In favor of https://github.com/somleng/somleng-scfm/pull/86

**Pull request closed without merging**